File: lab-1/p2.c
// 2. Find area of circle. (pie*r*r) (A)

#include <stdio.h>

void main() {
    float r;
    const float pi = 3.14;

    printf("Enter the radius of the circle: ");
    scanf("%f", &r);

    float area = pi * r * r;

    printf("The area of the circle is: %.2f\n", area);
}

File: lab-4/P2.java
// 2. WAP to find a diameter from given area of circle. (A)

import java.util.Scanner;

public class P2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter area : ");
        double area = scanner.nextDouble();

        double radius = Math.sqrt(area / Math.PI);
        double diameter = 2 * radius;

        System.out.printf("%.2f\n", diameter);

        scanner.close();
    }
}
